src/util-logopenfile.c

```diff
@@ -256,6 +256,31 @@ SCConfLogOpenGeneric(ConfNode *conf,
     if (append == NULL)
         append = DEFAULT_LOG_MODE_APPEND;
 
+    /* JSON flags */
+    ConfNode *json_flags = ConfNodeLookupChild(conf, "json");
```

Comment:
should the resulting ptr be checked? Not sure if ConfNodeLookupChildValue allow for a null ptr?